services/profile.js

```diff
@@ -1,16 +1,16 @@
-const mongoose = require("mongoose");
-let Profile = mongoose.model("profiles");
+const Profile = require("../models/Profile");
+const dateToZodiac = require("./date");
 
 module.exports.registerProfile = async (userId) => {
-  console.log("in registering new profile")
+  console.log("in registering new profile");
   let profileToSave = new Profile({
     userId: userId,
   });
-  console.log("new profile ",profileToSave )
+  console.log("new profile ", profileToSave);
   try {
-    await profileToSave.save();
+    return await profileToSave.save();
   } catch (err) {
-    if (err.code == 11000) {
+    if (err.code === 11000) {
```

Comment:
i think here we don't need that message below unless frontend will use it. Sveta was talking about those, I think she said they don't really need them. WE can leave it for now though and change it when do the task for adding error codes